mule-extensions-api/src/main/java/org/mule/runtime/extension/api/runtime/parameter/CorrelationInfo.java

```diff
@@ -36,4 +37,9 @@
    * @return The correlation id of the {@link Event} currently being processed
    */
   String getCorrelationId();
+
+  /**
+   * @return the {@link GroupCorrelation} of the event regarding it's relationship with other events in the same group.
+   */
+  GroupCorrelation getGroupCorrelation();
```

Comment:
this should make revapi explode. Don't you need to add an api-ignore.json?

Reply:
There is no error when compiling. Should there be an error when there is a method addition?Will check with @pablokraan still

Reply:
It's will not fail because CorrelationInfo was added en version 1.1